basedtyping/runtime_checks.py

```diff
@@ -0,0 +1,20 @@
+from types import UnionType
+from typing import Sequence, cast
+
+
+def issubform(type1: type | UnionType, type2: type | UnionType) -> bool:
+    """``issubclass`` but works with a union as the first arg (like it does for the second arg)
+
+    for example:
+
+    >>> issubclass(int | str, object)
+    TypeError: issubclass() arg 1 must be a class
+    >>> issubtype(int | str, object)
+    True
+    """
```

Comment:
```suggestion
    """``issubclass`` but supports typeforms (type-time types)

    >>> issubclass(int | str, object)
    TypeError: issubclass() arg 1 must be a class
    
    >>> issubform(int | str, object)
    True
    
    >>> issubform(int | str, str)
    False
    """
```

Reply:
fixed the function name in the example & added a TODO to make it work on all subforms